.github/workflows/codeql-analysis.yml

```diff
@@ -33,9 +33,11 @@ jobs:
     steps:
     - name: Checkout repository
       uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+  # Note: If Python dependencies are added to this workflow in future, install with '-c constraints.txt'
 
     # Initializes the CodeQL tools for scanning.
     - name: Initialize CodeQL
+      # github/codeql-action/init pinned to commit for v3.24.5
```

Comment:
This is redundant.  There is already a comment for this on the next line.  All these generated comments should be removed.